content/spaces/backup-and-restore.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Backup and restore Upbound Spaces
+weight: 120
+description: Enable and manage backups in your Upbound Space.
+---
+
+{{< hint "important" >}}
+This functionality is in preview and requires Spaces `v1.3.0`.
+{{< /hint >}}
+
+
+Upbound allows you to configure a backup and restore process. This feature helps manage your important control plane data and ensures security and availability best practices.
+
+## Usage
+
+### Backup
+
+#### Create a backup secret
+
+Before you begin the backup process, you need to create a secret for your Kubernetes configuration to allow for communication between your backup and your restored configuration.
+
+```bash
+KUBECONFIG=/tmp/ctp.yaml kubectl create secret generic super-secret-secret -n default --from-literal=password=supersecret
+```
+
+#### Single backup
```

Comment:
I think the flow should be to first create a `SharedBackupConfig`, then make a schedule, then discuss how to do a manual backup, then do a restore